.github/CODEOWNERS

```diff
@@ -71,7 +71,7 @@
 /texttospeech/**/*                     @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/python-samples-reviewers
 /translate/**/*                        @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/python-samples-reviewers
 /video/transcoder/*                    @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/python-samples-reviewers
-/bigquery-connection/**/*                    @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/python-samples-reviewers
+/bigquery-connection/**/*              @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/python-samples-reviewers
```

Comment:
Please add `/bigquery-datatrasfer/**/*` to CODEOWNERs file.

Reply:
Done.  Thanks!